README.rst

```diff
@@ -125,6 +125,22 @@ The default keys and values are specified below. If you want to use custom confi
     api = ADDRESS_OF_THE_SERVER # example: http://fancyUrlName.com:8080
     
 
+System failures
+~~~~~~~~~~~~~~~
+System failures are part of any software system. Upon some certain outputs (e.g. receiving error 500 from remote pipeline),
+we return :code:`None` in the output of call. When processing big documents it might make sense to check take care of
```

Comment:
If error occur, the variable will be not defined. Because an exception will be raised and ends the operation.
```python
>>> dc = p.doc("Hello")
Exception: You reached maximum query limit with default remote server (100 queries/day)
>>> print dc
NameError: name 'dc' is not defined
```